package.json

```diff
@@ -42,7 +42,7 @@
     "@nrwl/vite": "^15.6.3",
     "@nrwl/web": "15.6.3",
     "@nrwl/workspace": "15.6.3",
-    "@tacc/core-styles": "github:TACC/Core-Styles#675e5d5",
+    "@tacc/core-styles": "github:TACC/Core-Styles#task/accent-color-from-purple-to-blue",
```

Comment:
This is for testing. I'll point this to a commit, or—better—a version release, of Core-Styles before launch.